proxy/src/coco.rs

```diff
@@ -550,14 +550,15 @@ pub fn default_config(
 }
 
 #[cfg(test)]
+#[allow(clippy::panic)]
 mod test {
     use librad::keys::SecretKey;
 
     use crate::error::Error;
 
     #[tokio::test]
     async fn test_can_create_user() -> Result<(), Error> {
-        let tmp_dir = tempfile::tempdir().unwrap();
+        let tmp_dir = tempfile::tempdir().expect("failed to create temdir");
```

Comment:
You can avoid this by adding `clippy::unwrap_used`.

Reply:
Ah the error messages are useful at the same time :shrug: